sqlalchemy_ibmi/base.py

```diff
@@ -1042,7 +1063,6 @@ def get_indexes(self, connection, table_name, schema=None, **kw):
         return [value for key, value in indexes.items()]
 
     @reflection.cache
-    def get_unique_constraints(self):
+    def get_unique_constraints(self, connection, table_name, schema=None, **kw):
         unique_consts = []
         return unique_consts
-
```

Comment:
Is this an extra line or the trailing newline?

Reply:
trailing new line

Reply:
looks like this was an extra line that wasn't needed, not the necessary final newline.